tests/sample_data/experimental/test_run_recipe.py

```diff
@@ -23,11 +29,26 @@ def recipe():
 
 @pytest.mark.use_sample_data
 def test_run_recipe(recipe, tmp_path):
-    """Test running a basic recipe using sample data."""
+    """Test running a basic recipe using sample data.
+
+    Recipe contains no provenance and no diagnostics.
+    """
     CFG['output_dir'] = tmp_path
 
     assert isinstance(recipe, Recipe)
 
     output = recipe.run()
 
-    assert not output  # output is not yet defined
+    assert len(output) > 0
+    assert isinstance(output, RecipeOutput)
+
+    for task, task_output in output.items():
+        assert isinstance(task_output, TaskOutput)
+        assert len(task_output) > 0
```

Comment:
If this recipe doesn't contain diagnostics, shouldn't the task output be empty?

Reply:
Apparently it returns the preprocessor files.